lib/inbox/pages/inbox_page.dart

```diff
@@ -102,19 +102,28 @@ class _InboxPageState extends State<InboxPage> {
         ],
         bottom: PreferredSize(
           preferredSize: const Size.fromHeight(45.0),
-          child: Padding(
-            padding: const EdgeInsets.only(bottom: 8.0),
-            child: Center(
-              child: InboxCategoryWidget(
-                inboxType: inboxType,
-                onSelected: (InboxType? selected) {
-                  _scrollController.animateTo(0, duration: const Duration(milliseconds: 150), curve: Curves.easeInOut);
-                  setState(() {
-                    inboxType = selected;
-                  });
-                },
-              ),
-            ),
+          child: BlocBuilder<InboxBloc, InboxState>(
+            builder: (context, state) {
+              return Padding(
+                padding: const EdgeInsets.only(bottom: 8.0),
+                child: Center(
+                  child: InboxCategoryWidget(
+                    inboxType: inboxType,
+                    onSelected: (InboxType? selected) {
+                      _scrollController.animateTo(0, duration: const Duration(milliseconds: 150), curve: Curves.easeInOut);
+                      setState(() {
+                        inboxType = selected;
+                      });
+                    },
+                    unreadCounts: {
+                      InboxType.replies: context.read<InboxBloc>().state.repliesUnreadCount,
```

Comment:
Just curious, but since you're using `BlocBuilder<InboxBloc, InboxState>`, can you retrieve the `repliesUnreadCount` from `state.repliesUnreadCount`?

If so, this will reduce any extra calls needed to look up `InboxBloc`!

Reply:
Yep, that seems to work! Just pushed!